cozy.example.yaml

```diff
@@ -318,10 +318,16 @@ contexts:
     init_photos_folder: true
     # Allows to override the default template "Cozy" title by your own title
     templates_title: "My Personal Cloud"
-    # Use a different noreply mail for this context
-    noreply_address: noreply@cozy.beta
-    noreply_name: My Cozy Beta
-    reply_to: support@cozy.beta
+    # Use a different config for mails for this context
+    mail:
+      # If the host is set to "-", no mail will be sent on this context
+      host: smtp.cozy.beta
+      port: 465
+      username: {{.Env.COZY_BETA_MAIL_USERNAME}}
+      password: {{.Env.COZY_BETA_MAIL_PASSWORD}}
+      noreply_address: noreply@cozy.beta
```

Comment:
moving this from context.name to context.name.mail should keep backward compatibility for older config scheme

Reply:
It is the case: the stack looks at both `context.name.noreply_address` and `context.name.mail.noreply_address` for example. I have documented only the new way, but there is no need to change the current configuration.

Reply:
Oh, I didn't saw that. Thanks.